packages/session-replay-browser/src/session-replay.ts

```diff
@@ -285,10 +308,22 @@ export class SessionReplay implements AmplitudeSessionReplay {
         }
         const eventString = JSON.stringify(event);
         const deviceId = this.getDeviceId();
-        deviceId && this.eventsManager && this.eventsManager.addEvent({ event: eventString, sessionId, deviceId });
+        this.eventsManager &&
+          deviceId &&
+          this.eventsManager.addEvent({ event: { type: 'rrweb', data: eventString }, sessionId, deviceId });
       },
       packFn: pack,
       inlineStylesheet: this.config.shouldInlineStylesheet,
+      hooks: {
+        mouseInteraction:
+          this.eventsManager &&
+          clickHook({
+            getGlobalScopeFn: getGlobalScope,
```

Comment:
Why pass this as an option? Is it for testing reasons? If so I can point you to where I've mocked global scope for other testing purposes

Reply:
Good callout. I had thought this was an instance function is probably why I added it as a parameter.